content/api/inline-seeds.apib

```diff
@@ -0,0 +1,511 @@
+
+FORMAT: 1A
+title: Automatic Inline Seeding
+description: Accept campaign stats, sanitize PII, and trigger seed messages.
+label: Deliverability
+# Group Automatic Inline Seeding API
+
+The Automatic Inline Seeding API helps customers automatically send messages from their campaigns to seed addresses. This document is for customers that are sending with SparkPost. Customers that are sending using another offering should see TODO: link here to on-prem docs.
+
+## Seed Config
+
+Seeding options that will apply to campaigns that match the provided rules.
+
++ Data Structure
+  + threshold (number) - Number of messages seen for a campaign before seed sending starts. <br /> Maximum: `1,000,000`<br /> Minimum: `1,000`
+    + Default: 10,000
+  + duration (number) - Number of minutes after threshold is met over which to spread out seed messages. <br /> Maximum: `720` (12 hours)<br /> Minimum: `20`
+    + Default: 30
+  + reset (number) - Number of hours after threshold is met until seed sending can happen again. <br /> Maximum: `168` (7 days) <br /> Minimum: `22`
+    + Default: 24
+  + match (array) - Array of campaign ids or prefixes that are eligible. Strings ending with an asterisk are treated as prefixes. <br /> Maximum: `5` entries.
+  + exclude (array) Array of campaign ids or prefixes that are NOT eligible. Strings ending with an asterisk are treated as prefixes. <br /> Maximum: `5` entries.
+
+### Get Seed Config [GET /inline-seeds/config]
+
+Returns config details for the authenticated account.
+Only one seed config with 5 match/exclude rules max is currently supported.
+One or the other of (match, exclude) may be used, not both.
+Auth context (from API key) provides customer and subaccount ids.
+
++ Request
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
++ Response 200 - No matches (disabled) (application/json)
+
+    + Body
+
+        ```json
+        {
+          "results": {}
+        }
+        ```
+
++ Response 200 - Both types of matches (application/json)
+
+    + Body
+
+        ```json
+        {
+          "results": {
+            "configs": [
+              {
+                "threshold": 1000,
+                "duration": 20,
+                "reset": 24,
+                "match": [
+                  "my-campaign",
+                  "my-*"
+                ]
+              }
+            ]
+          }
+        }
+        ```
+
+### Create Seed Config [POST /inline-seeds/config]
+
+Create seed config for the authenticated account.
+
++ Request - Create a seed config with match rule (application/json)
```

Comment:
We should remove the `-` here because it's appearing in the rendered docs like this:
>Request: - Create a seed config with match rule